rails-tasklist/app/views/people/list.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="container">
+  <h1><%= @person.name %></h1>
+    <table class="table table-hover">
+      <tr >
+        <th>
```

Comment:
You could save a lot of whitespace by putting the opening and closing tags on the same line together since there is no content.